openstack/orchestration/v1/apiversions/requests.go

```diff
@@ -0,0 +1,13 @@
+package apiversions
+
+import (
+	"github.com/rackspace/gophercloud"
+	"github.com/rackspace/gophercloud/pagination"
+)
+
+// ListVersions lists all the Neutron API versions available to end-users
+func ListVersions(c *gophercloud.ServiceClient) pagination.Pager {
```

Comment:
For API version functionality, do you think it's better to shift it up a level to the `openstack/orchestration/apiversions` namespace?